ewutils.py

```diff
@@ -1365,3 +1362,97 @@ def check_fursuit_active(id_server):
 		return True
 	else:
 		return False
+
+def create_death_report(cause = None, user_data = None):
+	
+	client = ewcfg.get_client()
+	server = client.get_server(user_data.id_server)
+
+	# User display name is used repeatedly later, grab now
+	user_member = server.get_member(user_data.id_user)
+	user_nick = user_member.display_name
+
+	report_requires_killer = [ewcfg.cause_killing, ewcfg.cause_busted, ewcfg.cause_burning, ewcfg.cause_killing_enemy]
+	if(cause in report_requires_killer): # Only deal with enemy data if necessary
+		
+		killer_isEnemy = False
+		killer_isUser = False
+
+		# Generate list of enemy inflicted traumas
+		enemy_trauma_list = []
+		for atype in ewcfg.enemy_attack_type_list:
+			enemy_trauma_list.append(atype.id_type)
+
+		# Enemies have unique traumas, meaning if they have enemy trauma the killer was an enemy
+		if(user_data.trauma in enemy_trauma_list):
```

Comment:
this is such an ass backwards way of checking this. can't we just check the cause of death instead?

Reply:
yup. sure can. Guess I was thinking too much on that one